tox.ini

```diff
@@ -58,7 +58,6 @@ passenv =
     dev: CLIENT_RPC_ALLOWED_ORIGINS
     dev: CLIENT_URL
     dev: GOOGLE_ANALYTICS_TRACKING_ID
```

Comment:
Are we keeping this one? What's the difference? tracking on the client vs in H itself?

Reply:
>  tracking on the client vs in H itself?

Yes, this is the distinction. `GOOGLE_ANALYTICS_TRACKING_ID` is for analytics on pages served by h. We don't look at GA for these pages often, but it is still enabled currently.